### ISSUE TITLE
Karazhan Side Entrance Keys

### ISSUE THREAD

**Issue by rmzV** [cmangos/wotlk-db#905]
according to this
https://us.forums.blizzard.com/en/wow/t/karazhan-side-entrance-keys/614916
"Hi There, I was running karazhan a few times and noticed that Moroes doesn’t drop the Set of Rusty Keys any more. Is this the case? Or am I missing something? How can I use the side entrance? Many thanks for any info you can provide."

Does the ingame NPC drops keys on server?

couse cant find them

**Comment by evil-at-wow** [comment]
No, that post is no doubt about the Legion dungeon version Return to Karazhan, not the TBC Karazhan.

**Comment by rmzV** [comment]
So keys must be dropped?
